app/celer-sim/celer-sim.cc

```diff
@@ -87,6 +87,12 @@ void run(std::istream* is, std::shared_ptr<OutputRegistry> output)
 #else
     CELER_ASSERT_UNREACHABLE();
 #endif
+    TracingSession tracing_session{
+        run_input->tracing_file.empty()
+            ? TracingSession{}
+            : TracingSession{run_input->tracing_file}};
```

Comment:
Maybe the `tracing_file` run input should have the default parameter (if perfetto is active) or the constructor could be simplified?

Reply:
I moved that check inside the constructor, a bit more ergonomic to use that way